omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -288,7 +288,7 @@ def generate_hash
       results["private_chef"]["logs"]["log_retention"] = PrivateChef["log_retention"]
       results["private_chef"]["logs"]["log_rotation"] = PrivateChef["log_rotation"]
       results["private_chef"]["dark_launch"] = PrivateChef["dark_launch"]
-      results["private_chef"]["opscode-erchef"]["max_request_size"] = PrivateChef["opscode_erchef"]["max_request_size"]
+#      results["private_chef"]["opscode-erchef"]["max_request_size"] = PrivateChef["opscode_erchef"]["max_request_size"]
```

Comment:
This doesn't seem related?

Reply:
This is needed for chef 15; issue fixed a different way in the end.